### DISCUSSION TITLE
Why do amssymb symbols not show up in tblr table?

### DISCUSSION THREAD

**Discussion by llarsen71** [lvjr/tabularray#378] [Q&A]
I am trying to include the **\Box** and **\boxtimes** symbols in a table. I am able to with  **tabular** tables, but not with **tblr**. What do I have to do to get these symbols to show up in a **tblr** table? Below is an example with the **tblr** table followed by the **tabular** table.  An image of what is generated from lualatex is shown afterward:

```
\documentclass[letterpaper,10pt]{article}
\usepackage{tabularray}
\usepackage{amssymb}                                          % Symbols \Box and \boxtimes

%===============================================================================
\begin{document}
%===============================================================================

\begin{tblr}{ll}
\hline
\Box~~yes & \boxtimes~~no \\
\hline
\end{tblr}

\vspace{0.2in}

\begin{tabular}{ll}
\hline
\Box~~yes & \boxtimes~~no \\
\hline
\end{tabular}

\end{document}
```

![image](https://user-images.githubusercontent.com/1498201/223219863-786e06c9-2653-4b92-8e2e-72dda8b8796b.png)

**Comment by davidcarlisle** [comment]
unrelated to `tblr` they are math symbols, you get

`! Missing $ inserted.`

so you need

`$\Box$~~yes & $\boxtimes$~~no \\ `

in `tabular` and `ltblr`

**Comment by llarsen71** [comment]
Interesting! That makes me wonder why this worked in the tabular environment.

**Reply by davidcarlisle** [reply]
@llarsen71  it does not work. remove `tblr` from your test file and you get the same error.

**Comment by FrankMittelbach** [comment]
well it wouldn't, would it? (if it would you must have loaded some package that redefines them to use \ensuremath (probably).
